### ISSUE TITLE
New #pragma cling: unload

### ISSUE THREAD

**Issue by ax3l** [root-project/cling#205]
Hi,

is it possible to add the `#pragma load` equivalent for `.U` to the list of supported pragmas for consistency? :)
Probably needs to be added in `cling/lib/Interpreter/ClingPragmas.cpp`.

cc @SimeonEhrig

**Comment by Axel-Naumann** [comment]
I'd rather want to invest my time on making unloading sturdier rather than making it more accessible...

**Comment by ax3l** [comment]
But we need unloading a lot on Jupyter notebooks to overwrite things, don't we? :)

**Comment by Axel-Naumann** [comment]
Indeed - which is why I'd rather want to fix that first!

**Comment by ax3l** [comment]
Yes, we (cc @SimeonEhrig) realized in a simple demonstration that this pragma would actually be quite useful for quick tests.
